src/kbmod/work_unit.py

```diff
@@ -205,8 +253,27 @@ def from_fits(cls, filename):
 
                 imgs.append(LayeredImage(sci, var, msk, p))
 
+            per_image_wcs = []
+            per_image_ebd_wcs = []
+            constituent_images = []
+            for i in range(n_constituents):
+                # Extract the per-image WCS if one exists.
+                per_image_wcs.append(extract_wcs_from_hdu_header(hdul[f"WCS_{i}"].header))
+                per_image_ebd_wcs.append(extract_wcs_from_hdu_header(hdul[f"EBD_{i}"].header))
```

Comment:
Same comment as the helio/geo distances in terms of EBD_{i}. Should it be optional for unreprojected images?